egov-persister/hearing-persister.yml

```
@@ -130,4 +130,23 @@ serviceMaps:
+      objecIdJsonPath: $.id
+      tenantIdJsonPath: $.tenantId
+      auditAttributeBasePath: hearings.*
+      queryMaps:
+        - query: UPDATE dristi_hearing SET startTime = ?, endTime = ?, lastModifiedBy = ?, lastModifiedTime = ? WHERE hearingId = ?;
+          basePath: hearings.*
+          jsonMaps:
+            - jsonPath: $.hearings.*.startTime
+            - jsonPath: $.hearings.*.endTime
+            - jsonPath: $.hearings.*.auditDetails.lastModifiedBy
+            - jsonPath: $.hearings.*.auditDetails.lastModifiedTime
+            - jsonPath: $.hearings.*.hearingId
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)